api/test/jest.setup.js

```diff
@@ -1,4 +1,6 @@
 global.beforeAll(() => {
   global.API_BASE = 'http://localhost:8081/api/v2';
   global.HUMAN_GEM_VERSION = '1_10_0';
```

Comment:
Is that possible to define the VERSION for GEMs from e.g. `api/src/data/integratedModels.json`? Otherwise, we may forget to update them when we are updating the models.

Reply:
Good point! Added in afe2cc5